### ISSUE TITLE
Android Firefox 68.0 does not support encoding for webaudiofont

### ISSUE THREAD

**Issue by sdoherty0** [surikov/webaudiofont#35]
Firefox 68.0 on Android phone will no longer play webaudiofont midi files. Even your simple sample https://surikov.github.io/webaudiofont/examples/simple.html# throws the following error. It is this error for my site as well.

The buffer passed to decodeAudioData contains invalid content which cannot be decoded successfully.
EncodingError: The given encoding is not supported.

I don't think it matters which tone file is loaded as your sample loads a different one from my site. Android Chrome works fine and so do desktop (PC) Chrome and Firefox. It is only Android 68.0 Firefox that is throwing this error - as well as Android FF Beta, and Developer's Nightly. So it seems like new mobile Firefox will be incompatible with something in your code base.

Do you have any insight into what could be tweaked to get this working again?

**Comment by yw662** [comment]
https://github.com/surikov/webaudiofont/blob/6aa83c160256f542310fb9b9c4676ac4ada4dd52/npm/src/player.js#L262

The buffer is copied byte by byte from the font.
https://github.com/surikov/webaudiofont/blob/6aa83c160256f542310fb9b9c4676ac4ada4dd52/npm/src/player.js#L258-L261

Which means, this **is** related to individual sound fonts.
However, it seems that most of the sound fonts are unsupported.

However, it is really wired that ff android does not supported it :-). This issue should be more about android and ff, than webaudiofont, indeed.

**Comment by surikov** [comment]
I think Android Firefox 68.0 broke some libraries for decoding of audio.
WebAudioFont uses different compression for different instruments.

- open https://surikov.github.io/webaudiofont/examples/dynamic.html
- select instrument 7.Accousic Grand Piano
- press Start

this is uncompressed sample.

**Comment by sdoherty0** [comment]
https://surikov.github.io/webaudiofont/examples/dynamic.html using instrument 7.Accousic Grand Piano does work successfully. So this is related to the selection of instruments?

**Comment by surikov** [comment]
I think Android Firefox 68.0 broke some libraries for decoding of audio.
WebAudioFont uses different compression for different instruments.

**Comment by kbrosnan** [comment]
@sdoherty0 What device and OS are you using?

**Comment by sdoherty0** [comment]
I am using a Samsung  Galaxy Express Prime running Android 9.0. It is funny because my wife's LG has FF 68.0 and Andriod 8 something and it works. And a friend who is running Andriod 8.1 has FF 68.0 and it works on his phone. Chrome works on my phone. I have uninstalled and reinstalled FF. I have installed the Beta and the Nightly and it fails on all of those on my phone. It might be a way that FF 68.0 accesses some underlying library which might have changed in Android 9.0. But that is weird because Chrome works. So, I'm not sure what the source of this problem is exactly.
